src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -47,70 +47,82 @@ NRQL requirements:
 4. Enter the New Relic export account ID: `888632727556`.
 5. Select <DNT>**Require external ID**</DNT> and enter the [account ID](/docs/accounts/accounts-billing/account-structure/account-id) of the New Relic account you want to export from.
 6. Click <DNT>**Permissions**</DNT>, and then select the policy you created above.
 7. Add a role name (this will be used in the export registration) and description.
 8. Create the role.
-
-When that's done, you can work on setting up your export rules using NerdGraph. For more on that, keep reading.
-
+  </Step>
+</Steps>
+                     
+When you're done with these steps, you can set up your export rules using NerdGraph. For more on that, jump to [Important fields for NerdGraph calls](#fields).
+   
 
 ## Set up an Azure Event Hub [#event-hub-setup]
 
 To set up streaming data export to Azure, you must first set up an Event Hub. We'll walk you through that procedure in the next three steps.
 
 Alternatively, you can follow the Azure guide [here](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
 
-### Step 1. Create an Event Hubs namespace [#create-event-hubs-namespace]
+<Steps>
+  <Step>
+### Create an Event Hubs namespace [#create-event-hubs-namespace]
 
 1. Navigate to Event Hubs within your Microsoft Azure account.
 2. Follow the steps to create an Event Hubs namespace. We recommend enabling auto-inflate to ensure you receive all of your data.
 3. Ensure public access is enabled, we will be using a Shared Access Policy to authenticate securely with your Event Hub.
 4. Once your Event Hubs namespace is deployed, click <DNT>**Go to resource**</DNT>.
-
-### Step 2. Create an Event Hub[#create-event-hub]
+  </Step>
+  <Step>
+### Create an Event Hub[#create-event-hub]
 
 1. In the left column, click <DNT>**Event Hubs**</DNT>.
 2. Then click <DNT>**+Event Hub**</DNT> to create an Event Hub.
 2. Enter the desired Event Hub Name. Save this, as you'll need it later to create the streaming export rule.
 4. Once the Event Hub is created, click on the Event Hub.
-
-### Step 3. Create and attach a shared access policy [#event-hub-policy]
+  </Step>
+  <Step>
+### Create and attach a shared access policy [#event-hub-policy]
 
 1. In the left column, go to <DNT>**Shared access policies**</DNT>.
 2. Click <DNT>**+Add**</DNT> near the top of the page.
 3. Choose a name for your shared access policy.
 4. Check <DNT>**Send**</DNT>, and click <DNT>**Create**</DNT>.
 5. Click the created policy, and copy the <DNT>**Connection string–primary key**</DNT>. Save this, as we'll use this to authenticate and send data to your Event Hub.
+  </Step>
+</Steps>
 
-When that's done, you can work on setting up your export rules using NerdGraph. For more on that, keep reading.
+When you're done with these steps, you can set up your export rules using NerdGraph. For more on that, jump to [Important fields for NerdGraph calls](#fields).
```

Comment:
@rhetoric101 I don't think you need this link, because the thing you're linking to is literally the next line.

Reply:
OK, I removed your link and replaced it with wording that was similar to yours.